app/views/admin/merchants/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_with url: admin_merchants_path, method: :post, local: true do |form| %>
+  <%= form.label :name %>
+  <%= form.text_field :name %>
+
+  <%= form.hidden_field :status, value: 'disabled' %>
```

Comment:
Is this hidden field here to create a default value for status? I think a better way would be to simply set a default value in the schema, that way we don't have to worry about assigning it or hidden fields.

Reply:
Why yes.... yes, it was! I made those changes to set disabled as a default. I meant to include a sad path when I made these updates but forgot until just this moment. I will make a note to myself and will definitely add that!